### PR TITLE
[Github Actions] - Choose RC Commit Workflow

### PR DESCRIPTION
**Pull request by benWize**
As part of the [Beam Release Process](https://beam.apache.org/contribute/release-guide/), the following PR includes the workflow to run the [choose_rc_commit.sh](https://github.com/apache/beam/blob/master/release/src/main/scripts/choose_rc_commit.sh) script described in [step 8. Tag a chosen commit for the RC](https://beam.apache.org/contribute/release-guide/#tag-a-chosen-commit-for-the-rc)

Files:
- Choose RC Commit: .github/workflows/choose_rc_commit.yml
- Updated CI.md file

------------------------

Thank you for your contribution! Follow this checklist to help us incorporate your contribution quickly and easily:

 - [ ] [**Choose reviewer(s)**](https://beam.apache.org/contribute/#make-your-change) and mention them in a comment (`R: @username`).
 - [ ] Mention the appropriate issue in your description (for example: `addresses #123`), if applicable. This will automatically add a link to the pull request in the issue. If you would like the issue to automatically close on merging the pull request, comment `fixes #<ISSUE NUMBER>` instead.
 - [ ] Update `CHANGES.md` with noteworthy changes.
 - [ ] If this contribution is large, please file an Apache [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

See the [Contributor Guide](https://beam.apache.org/contribute) for more tips on [how to make review process smoother](https://beam.apache.org/contribute/get-started-contributing/#make-the-reviewers-job-easier).

To check the build health, please visit [https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md](https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md)

GitHub Actions Tests Status (on master branch)
------------------------------------------------------------------------------------------------
[Build python source distribution and wheels](https://github.com/apache/beam/actions?query=workflow%3A%22Build+python+source+distribution+and+wheels%22+branch%3Amaster+event%3Aschedule)
[Python tests](https://github.com/apache/beam/actions?query=workflow%3A%22Python+Tests%22+branch%3Amaster+event%3Aschedule)
[Java tests](https://github.com/apache/beam/actions?query=workflow%3A%22Java+Tests%22+branch%3Amaster+event%3Aschedule)
[Go tests](https://github.com/apache/beam/actions?query=workflow%3A%22Go+tests%22+branch%3Amaster+event%3Aschedule)

See [CI.md](https://github.com/apache/beam/blob/master/CI.md) for more information about GitHub Actions CI.

### CODE REVIEW

File: .github/workflows/choose_rc_commit.yml [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+        required: false
+        default: no
+      OVERWRITE:
+        description: Overwrite RC Tag (yes/no)
+        required: false
+        default: no
+      DEBUG:
+        description: Enable debug flag adding --debug
+        required: false
+        default: ""
```
Comment (damccorm) [inline .github/workflows/choose_rc_commit.yml:47]: It was unclear from the description that "--debug" and "" are the only valid inputs. Could we change this to a (yes/no) option like the other inputs?

File: .github/workflows/choose_rc_commit.yml [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+  workflow_dispatch:
+    inputs:
+      RELEASE:
+        description: Beam version of current release
+        required: true
+      RC:
+        description: RC version for the release
+        required: true
+      COMMIT:
+        description: Selected commit to create the release
```
Comment (damccorm) [inline .github/workflows/choose_rc_commit.yml:34]: For all of these descriptions, could you add an example?
```suggestion
        description: Beam version of current release (e.g. 2.XX.0)
        required: true
      RC:
        description: Integer RC version for the release (e.g. 3 for RC3)
        required: true
      COMMIT:
        description: Selected commit to create the release (should be full commit sha)
```

I'm not sure if my suggested descriptions are correct, so please validate. For example, should it be 3 or RC3 for the `RC` input?

**Comment by benWize** [comment]
Thanks, @damccorm, I applied your suggestions

**Review by damccorm — APPROVED** [review]
LGTM, thanks!

**Pull request merged**